pwa/design.md

```diff
@@ -1,5 +1,7 @@
 # Design Decisions
 
+After user submits a new form, they are redirected to the edit form for the created source entry.
```

Comment:
Better. However, I do have some feedback.

There is **already** a section called _Decision Log_, this is where this should be placed. So underneath the `React Hook Form` and `React Query` decisions, you can add the following decision:

- Form redirects throughout the application are based on the action a user takes. On a _create_, the user is always redirected to the detail page of the created resource, on an _update_ the user is always redirected back to the collection page.